controller/system_rollout_controller.go

```diff
@@ -1215,9 +1202,16 @@ func (c *SystemRolloutController) restorePersistentVolumeClaims() (err error) {
 	}
 
 	for _, restore := range c.persistentVolumeClaimList.Items {
+		isSkippedByDependency, err := c.hasSystemRestoredAnnotation(&restore, true)
```

Comment:
Suggest renaming `hasSystemRestoredAnnotation` -> `isResourceRestoreRequired` as below. (no need to pass a bool param, because this will make the code not easy to read. )

```go

restoreRequired, err := c.isResourceRestoreRequired(&restore)
```